src/controllers/statisticsController.js

```diff
@@ -0,0 +1,104 @@
+import models from '../db/models';
+
+const { Statistics } = models;
+
+/**
+ * @description class responsible for statistics of author's article readCount
+ *
+ * @class StatisticsController
+ */
+class StatisticsController {
+  /**
+   * @description - This method is responsible for counting updating article readCount
```

Comment:
@marcdomain, can you correct the `@description` of the `StatisticController`. It reads: 
> This method is responsible for counting updating article readCount
Thanks

Reply:
Thank you. It will be corrected